plugins/source-kafka/README.md

```diff
@@ -53,26 +53,26 @@ Manage your Knative Kafka eventing sources
 
 #### SEE ALSO
 
-* [kafka create](#kafka-create)	 - create NAME
-* [kafka delete](#kafka-delete)	 - delete NAME
-* [kafka describe](#kafka-describe)	 - describe NAME
+* [kafka create](#kafka-create)	 - Create a kafka source
+* [kafka delete](#kafka-delete)	 - Delete a Kafka source
```

Comment:
Sorry but still seeing "Delete a Kafka source" which is not consistent with the "kafka" spelling. Am I missing something?

Reply:
change all command output to lower case, except the first word of a sentence.